File: Fibonacci.py
def fibonacci(n):
    if n <= 1: return n
    else: return fibonacci(n - 1) + fibonacci(n - 2)

nums = int(input("Fibonacci numbers to show: "))

if nums <= 0: print("Must be a positive integer")
else:
    print("Fibonacci series: ")
    for i in range(nums): print(fibonacci(i), end = ' ')

File: Fibonacci.c
#include <stdio.h>

int main() {
   int n, i, t1 = 0, t2 = 1, nextVal;
   printf("Fibonacci numbers to show: ");
   scanf("%d", &n);
   if (n <= 0) printf("Must be a positive integer");
   else {
       printf("Fibonacci series:\n");
       for (i = 1; i <= n; ++i) {
            printf("%d ", t1);
            nextVal = t1 + t2;
            t1 = t2;
            t2 = nextVal;
        }
   }
   printf("\n");
   return 0;
}
